stingray/io.py

```diff
@@ -653,3 +654,30 @@ def get_btis(gtis, start_time=None, stop_time=None):
 def gti_len(gti):
     """Return the total good time from a list of GTIs."""
     return np.sum([g[1] - g[0] for g in gti])
+
+
+def save_object(object, filename):
+    """
+    Pickle a class instance.
+
+    Parameters
+    ----------
+    object: a class instance
+    filename: str (name of the file to be created)
```

Comment:
A better way to write docstrings is.
```
Parameters
----------
x : type
    Description of parameter `x`.
```
This is also recommended at most places.